docs/contribution.adoc

```diff
@@ -0,0 +1,26 @@
+== Contribution
+
+For contributing to the API **docker** and the cloned **code-base** is required. 
+One of the dependencies to run the API is a Stargate-Container which represents the database. This service needs to run next to the actual API. +
+
+There are multiple ways to make the contribution
+
+* From your local development-environment
+* From the cloud with **Gitpod**
+
+=== Local Environment
+When running the API on your local machine, its required to run docker.
+
+1. Clone the Repo
+2. Install your dependencies with `npm install`
+3. Run `docker-compose up` to boot the Stargate-Container
+4. As soon docker-compose is completely booted you can start the API with `npm run start:dev`
+
+=== Gitpod
+
+Gitpod is a Service, which provides development-environments in the cloud. In case no plan is purchased the reguar `Opensource License` allows to use the service for 50 hours per month for free. +
```

Comment:
```suggestion
Gitpod is a Service, which provides development environments in the cloud. In case no plan is purchased the regular `Opensource License` allows using the service for 50 hours per month for free. +
```